docs/actions.md

```diff
@@ -607,7 +607,7 @@ You can create a OpenWhisk action called `helloSwift` from this function as
 follows:
 
 ```
-wsk action create helloSwift hello.swift
+wsk action create helloSwift hello.swift --kind swift:3.1.1
```

Comment:
I think that the first time we show how to create an action for a given language, we should use the default and not use `--kind`. (i.e. this should be the same as how JS and Python are shown.)